File: 프로그래머스/lv1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    grade = [6, 6, 5, 4, 3, 2, 1]
    answer = []
    correct = 0
    for i in lottos:
        if i in win_nums:
            correct += 1
    random = lottos.count(0)
    
    answer.append(grade[correct + random])
    answer.append(grade[correct])
    
    return answer

File: 프로그래머스/1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.js
function solution(lottos, win_nums) {
    const grade = [6, 6, 5, 4, 3, 2, 1]
    var answer = [];
    let unknown = 0;
    let correct = 0;
    for (let i = 0; i < lottos.length; i++) {
        const lotto = lottos[i]
        if (lotto === 0) {
            unknown++
            continue
        }
        
        if (win_nums.includes(lotto)) {
            correct++
        }
    }
    answer.push(grade[correct + unknown])
    answer.push(grade[correct])
    return answer;
}